src/browser-lib/types.ts

```diff
@@ -0,0 +1,19 @@
+import { NgZone, Type, NgModuleRef } from '@angular/core';
+import { AppProps } from 'single-spa';
+
+export interface SingleSpaAngularOpts {
```

Comment:
Why do we need this file? The current build process outputs `.d.ts` files (https://unpkg.com/browse/single-spa-angular@3.3.0/lib/browser-lib/single-spa-angular.d.ts). Can't we just keep these types in the file that uses them?

Reply:
No problem. I will these changes to stick in the single file.